pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -394,6 +394,10 @@ func (gce *GCECloud) EnsureTCPLoadBalancer(name, region string, externalIP net.I
 		PortRange:  fmt.Sprintf("%d-%d", minPort, maxPort),
 		Target:     gce.targetPoolURL(name, region),
 	}
+	if loadBalancerIP != nil {
+		req.IPAddress = loadBalancerIP.String()
```

Comment:
This.  I think my question is: Why isn't this entire change just the addition of:

```go
if externalIP != nil {
        req.IPAddress = externalIP.String()
}
```

... and nothing else?